elements/bulbs-video/elements/summary.test.js

```diff
@@ -71,8 +71,14 @@ describe('<bulbs-video-summary>', () => {
     it('renders <VideoSummaryView>', () => {
       subject = shallow(<BulbsVideoSummary nowPlaying=''/>);
       subject.setState({ video });
+
       expect(subject.equals(
-        <VideoSummaryView video={video} nowPlaying={true}/>
+        <VideoSummaryView
```

Comment:
Took another look at what I did here, turned out to be unnecessary, removed this change entirely.